ISAAR.MSolve.SamplesConsole/Examples/Static/Linear/Hexa8SimpleExample.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using ISAAR.MSolve.PreProcessor;
+using ISAAR.MSolve.Solvers.Skyline;
+using ISAAR.MSolve.Problems;
+using ISAAR.MSolve.Analyzers;
+using ISAAR.MSolve.Logging;
+using ISAAR.MSolve.PreProcessor.Materials;
+using ISAAR.MSolve.PreProcessor.Elements;
+using ISAAR.MSolve.Matrices;
+
+namespace ISAAR.MSolve.SamplesConsole.Examples.Static.Linear
+{
+    class Hexa8SimpleExample
+    {
+        private static IList<Node> CreateNodes()
+        {
+            IList<Node> nodes = new List<Node>();
```

Comment:
Be more concise by adding the nodes like this:

nodes.Add(new Node { ID = 1, X = 0.0, Y = 0.0, Z= 0.0 });